core/common/src/annotation/TextStyle.ts

```diff
@@ -182,6 +192,7 @@ export class TextStyleSettings {
     this.superScriptOffsetFactor = props.superScriptOffsetFactor ?? defaults.superScriptOffsetFactor;
     this.superScriptScale = props.superScriptScale ?? defaults.superScriptScale;
     this.widthFactor = props.widthFactor ?? defaults.widthFactor;
+    this.tabInterval = undefined === props.tabInterval ? defaults.tabInterval : props.tabInterval;
```

Comment:
nit: Could this use `??` operator like the pervious values for consistency? 

```suggestion
    this.tabInterval = props.tabInterval ?? defaults.tabInterval;
```